src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -117,6 +118,9 @@ def __init__(self) -> None:
         self._background_point = None
         self.worker_config = None
 
+        # declaration of instance variables
+        self._map_ptr = None # This is the map of the indices of input frame
```

Comment:
The fact that a comment is needed makes me think that `map_ptr` is perhaps not explicit enough as a name.

`frame_indices` ? `position_to_frame`?

Reply:
To me it is a `map of pointers` with minimal documentation about what it is.
Since I dislike formatter which are wrapping lines (making code unreadable IMHO), keeping variable names short limits such wrapping, thus the short variable names.

Reply:
> keeping variable names short limits such wrapping

There is no formatter run here so I don't understand your justification.

Anyway, if the variable name is explicit enough for you, good. My comments are and stay suggestions.